site.cfg.example

```diff
@@ -52,6 +52,26 @@
 #       True) to tell numpy.distutils to prefer static libraries (.a) over
 #       shared libraries (.so). It is turned off by default.
 #           search_static_first = false
+#
+#   runtime_library_dirs
+#       List of directories that contains the libraries that should be 
+#       used at runtime, thereby disregarding the LD_LIBRARY_PATH variable.
```

Comment:
It depends, you cannot override a set runtime_library_dir by
LD_LIBRARY_PATH. Only if the runtime library directory has not been set.
Hence, disregarding.

2015-02-25 10:10 GMT+01:00 Ralf Gommers <notifications@github.com>:

> In site.cfg.example
> <https://github.com/numpy/numpy/pull/5597#discussion_r25326065>:
>
> > @@ -52,6 +52,26 @@
> >  #       True) to tell numpy.distutils to prefer static libraries (.a) over
> >  #       shared libraries (.so). It is turned off by default.
> >  #           search_static_first = false
> > +#
> > +#   runtime_library_dirs
> > +#       List of directories that contains the libraries that should be
> > +#       used at runtime, thereby disregarding the LD_LIBRARY_PATH variable.
>
> it's more "removing the need to use" rather then "disregarding" right? One
> can still use a combination of both runtime_library_dirs and
> LD_LIBRARY_PATH if desired.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/numpy/numpy/pull/5597/files#r25326065>.
>



-- 
Kind regards Nick